third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -712,7 +721,7 @@ def arcs_kt_schema(
         name = name,
         srcs = outs,
         platforms = platforms,
-        deps = arcs_sdk_deps,
+        deps = arcs_sdk_deps + platform_deps,
         visibility = visibility,
     )
     outdeps = outdeps + arcs_sdk_deps
```

Comment:
Let's add `platform_deps` to `outdeps`. Doing so means we won't have to add `platform_deps` to `arcs_kt_gen` IIUC.

Reply:
Thinking about it more, this could use a refactor, something like: `deps` takes `outdeps`, and we set `outdeps` to `arcs_sdk_deps + platform_deps`.

Reply:
Thanks for this suggestion. I wasn't 100% following what was going on here, but it's making a little more sense now.